packages/block-library/src/navigation-menu/style.scss

```diff
@@ -106,4 +90,42 @@
 		}
 	}
 
+	// Menu event activation.
+	&.is-activated-by-hover li {
+		&:hover,
+		&:focus-within {
+			cursor: pointer;
+			z-index: 99999;
+		}
+
+		// Submenu Display
+		&:hover > ul,
+		&:focus-within > ul,
+		& ul:hover,
```

Comment:
Why are these needed? If the `ul` is within the `li` then the `:hover` on the `li` should sustain the hover effect.